src/wallet/wallet.cpp

```diff
@@ -999,6 +999,7 @@ bool CWallet::AddToWallet(const CWalletTx& wtxIn, bool fFlushOnClose)
     // Notify UI of new or updated transaction
     NotifyTransactionChanged(this, hash, fInsertedNew ? CT_NEW : CT_UPDATED);
 
+#if !(TARGET_OS_IPHONE || TARGET_IPHONE_SIMULATOR)
```

Comment:
Would prefer to have a `configure` check for `system` usability.

Reply:
Ok, I'll see if I can figure out how to do that.

Reply:
@luke-jr I made a separate PR for that #15457